src/reg-alloc.c

```diff
@@ -0,0 +1,242 @@
+static int liveness[N_RV_REGS];
+
+/* The priority queue of vm registers. The one which has farthest liveness is
+ * first.
+ */
+static uint8_t candidate_queue[N_RV_REGS];
+
+static inline void reset_liveness()
+{
+    for (int i = 0; i < N_RV_REGS; i++) {
```

Comment:
Rewrite with `memset`, which is optimized by C runtime. That is, `memset(liveness, 0xFF, sizeof(liveness))`.